src/coreclr/jit/emitarm64.cpp

```diff
@@ -18742,10 +18832,20 @@ void emitter::emitDispInsHelp(
 
         // <Pd>.<T>, <Pg>/Z, <Zn>.<T>, <Zm>.<T>
         case IF_SVE_CX_4A: // ........xx.mmmmm ...gggnnnnn.DDDD -- SVE integer compare vectors
+        case IF_SVE_GE_4A: // ........xx.mmmmm ...gggnnnnn.DDDD -- SVE2 character match
+        case IF_SVE_HT_4A: // ........xx.mmmmm ...gggnnnnn.DDDD -- SVE floating-point compare vectors
+            emitDispPredicateReg(id->idReg1(), PREDICATE_SIZED, id->idInsOpt(), true); // DDDD
+            emitDispPredicateReg(id->idReg2(), PREDICATE_ZERO, id->idInsOpt(), true);  // ggg
+            emitDispSveReg(id->idReg3(), id->idInsOpt(), true);                        // nnnnn
+            emitDispSveReg(id->idReg4(), id->idInsOpt(), false);                       // mmmmm
+            break;
+
+        // <Pd>.<T>, <Pg>/Z, <Zn>.<T>, <Zm>.D
+        case IF_SVE_CX_4A_A: // ........xx.mmmmm ...gggnnnnn.DDDD -- SVE integer compare vectors
             emitDispPredicateReg(id->idReg1(), PREDICATE_SIZED, id->idInsOpt(), true); // DDDD
             emitDispPredicateReg(id->idReg2(), PREDICATE_ZERO, id->idInsOpt(), true);  // ggg
             emitDispSveReg(id->idReg3(), id->idInsOpt(), true);                        // mmmmm
-            emitDispSveReg(id->idReg4(), id->idInsOpt(), false);                       // nnnnn
+            emitDispSveReg(id->idReg4(), INS_OPTS_SCALABLE_D, false);                  // nnnnn
```

Comment:
While passing `INS_OPTS_SCALABLE_D` looks fine for this instruction format, I was checking other formats where we directly pass the INS_OPTS and seems like for `IF_SVE_EQ_3A`, we should have a method that is reverse of `optWidenSveElemsizeArrangement()`, which will basically lower `H->B`, `S->H` and `D->S` instead of manipulating the `idOpts()` this way.

https://github.com/dotnet/runtime/blob/b4985820c0a9aac3fa2ad252cfda39e4f8a659f6/src/coreclr/jit/emitarm64.cpp#L18752-L18755

Reply:
Sure, makes sense. I will piggyback this change on my next patch 👍